public/app/features/plugins/sandbox/sandbox_plugin_loader.ts

```diff
@@ -130,31 +130,14 @@ async function doImportPluginModuleInSandbox(meta: PluginMeta): Promise<System.M
           }
         },
       }),
-      // This improves the error message output for plugins
-      // because errors thrown inside of the sandbox have a stack
-      // trace that is difficult to read due to all the sandboxing
-      // layers.
-      instrumentation: {
-        // near-membrane concept of "activity" is something that happens inside
-        // the plugin instrumentation
-        startActivity() {
-          return {
-            stop: () => {},
-            error: getActivityErrorHandler(meta.id),
-          };
-        },
-        log: () => {},
-        error: () => {},
-      },
     });
 
     // fetch plugin's code
     let pluginCode = '';
     try {
       pluginCode = await getPluginCode(meta);
     } catch (e) {
-      throw new Error(`Could not load plugin ${meta.id}: ` + e);
-      reject(new Error(`Could not load plugin ${meta.id}: ` + e));
+      reject(new Error(`Could not load plugin code ${meta.id}: ` + e));
```

Comment:
Intentional change. The existing code was not rejecting the promise and potentially keep the wait for a plugin load forever.